README.md

```diff
@@ -96,8 +96,6 @@ console.print("Where there is a [bold cyan]Will[/bold cyan] there [u]is[/u] a [i
 
 ![Console Markup](https://github.com/willmcgugan/rich/raw/master/imgs/where_there_is_a_will.png)
 
-You can use a Console object to generate sophisticated output with minimal effort. See the [Console API](https://rich.readthedocs.io/en/latest/console.html) docs for details.
```

Comment:
Hi! I'm sorry for jumping out of nowhere but was this removal accidental or intentional?

Reply:
No, was an error. Thanks. I'll restore it.